sections/data_rights.tex

```diff
@@ -12,3 +12,4 @@ \section{Data Rights to Alert Stream Components}\label{sec:data_rights}
 Instead, the LSST alert stream will be delivered to a to-be-determined set of community brokers (Section \ref{sec:community_brokers}).
 Any institution worldwide may propose to host a broker.
 Brokers may share (or not share) the contents of alert packets with whomever they choose, although the broker selection process (\S \ref{sec:community_brokers}) will prioritize selection of brokers that will provide world-public access to their services.
+\newtext{The contents of the \DIAObject, \DIASource, \DIAForcedSource, and \SSObject tables held in the PPDB are also public and may be freely shared, although the ability to access and query the PPDBs held in LSST DACs is restricted to data rights holders.}
```

Comment:
LSST DACs  in the USA and Chile is restricted ......

Reply:
(as we discussed on Slack the current text seems to cover this.)